protos/feast/core/JobService.proto

```diff
@@ -109,6 +113,12 @@ message StartOfflineToOnlineIngestionJobRequest {
 message StartOfflineToOnlineIngestionJobResponse {
     // Job ID assigned by Feast
     string id = 1;
+
+    // Job start time
+    google.protobuf.Timestamp job_start_time = 2;
+
+    // Feature table associated with the job
+    string feature_table = 3;
```

Comment:
Shouldn't naming be somehow in sync in request / response. In request there are two fields `project` & `table_name`. Is this one a `table_name`?